README.md

```diff
@@ -1,2 +1,242 @@
-# learn-user-testing
-Learn how to test your app with real humans 
+# Learn User Experience Testing
+Learn how to test your app with real humans
+
+## Why?
+This is _probably_ the most important/useful skill any "_creative technologist_"
+has. 💪
+
+Well conducted user experience testing helps you ensure that what you are
+planning to build is going to solve the problem for your users that you set out
+to solve. ✅
+
+It prevents you from wasting time ⏳/ effort 😓/ money 💰on guessing 🤔❓what your
+users will want, need or understand by getting them to critique your product as
+you go rather than retrospectively.
+
+Testing usability and following usability guidelines is not something we have to
+do to restrict ourselves or because we're told to. Understanding usability is
+about ensuring users can use our product effectively and so without it our
+product will be less successful. 📉👎  
+
+## What?
+User experience testing is a means of collecting feedback 🗣from users on the
+evolution of your product.
```

Comment:
Consider augmenting or revising this definition to include the fact that UX testing 
aims to get feedback on whether the UX/UI presented meets the needs of the user 
and allows them to achieve their desired goal in the fewest steps.

Reply:
Revised to:
>User experience testing is a means of collecting feedback 🗣from users on 
whether the UX/UI presented allows them to achieve their desired goal in the 
fewest steps.